src/test/java/org/cryptomator/cryptofs/fh/OpenCryptoFilesTest.java

```diff
@@ -16,12 +16,12 @@
 import java.nio.file.Path;
 import java.nio.file.Paths;
 
+import static org.mockito.ArgumentMatchers.any;
```

Comment:
Wrong import

Reply:
Fixed in https://github.com/cryptomator/cryptofs/pull/159/commits/62d7e9edf0c25d784983ce84a798c01369b84674